From f5339e32c9a06a43a18d0ebe48961c032ce178c0 Mon Sep 17 00:00:00 2001
From: Keian <52962697+IgnisWyvern@users.noreply.github.com>
Date: Wed, 7 Feb 2024 20:14:09 +0000
Subject: [PATCH] 60-evdev.hwdb: Add support for Huion Inspiroy 2 L (#31241)

Also fixed a spelling mistake on the HUION Inspiroy H420X

(cherry picked from commit e23d7aa1939fc17f5fa5ccf122fc7ecfe6386c03)

Related: RHEL-5950
---
 hwdb.d/60-evdev.hwdb | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hwdb.d/60-evdev.hwdb b/hwdb.d/60-evdev.hwdb
index 9a90420b5f..f657374986 100644
--- a/hwdb.d/60-evdev.hwdb
+++ b/hwdb.d/60-evdev.hwdb
@@ -482,8 +482,9 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnHP:pnHPENVYx360Convertible15-cn0xx
 # HUION
 #########################################
 
-# HUION Inpiroy H420X
+# HUION Inspiroy H420X and 2 L
 evdev:input:b0003v256Cp0064*
+evdev:input:b0003v256Cp0068*
  EVDEV_ABS_00=::200
  EVDEV_ABS_01=::200
 
